backend/benefit/applications/models.py

```diff
@@ -82,8 +92,40 @@ def _annotate_handled_at(self, qs):
         )
         return qs.annotate(handled_at=Subquery(subquery))
 
+    def _annotate_additional_information_requested_at(self, qs):
+        subquery = (
+            ApplicationLogEntry.objects.filter(
+                application=OuterRef("pk"),
+                to_status__in=[ApplicationStatus.ADDITIONAL_INFORMATION_NEEDED],
+            )
+            .order_by("-created_at")
+            .values("created_at")[:1]
+        )
+        return qs.annotate(additional_information_requested_at=Subquery(subquery))
+
+    def _annotate_submitted_at(self, qs):
+        subquery = (
+            ApplicationLogEntry.objects.filter(
+                application=OuterRef("pk"), to_status__in=[ApplicationStatus.RECEIVED]
+            )
+            .order_by("-created_at")
+            .values("created_at")[:1]
+        )
+        return qs.annotate(submitted_at=Subquery(subquery))
+
     def get_queryset(self):
-        return self._annotate_handled_at(super().get_queryset())
+        qs = super().get_queryset()
+        qs = self._annotate_with_log_timestamp(qs, "handled_at", self.HANDLED_STATUSES)
+        qs = self._annotate_with_log_timestamp(
+            qs,
+            "additional_information_requested_at",
+            [ApplicationStatus.ADDITIONAL_INFORMATION_NEEDED],
+        )
+        qs = self._annotate_with_log_timestamp(
```

Comment:
Just for clarification and not for renaming these fields but these are
 - "handled_last_at"
 - "additional_information_last_requested_at"
 - "last_submitted_at"
Right?

Reply:
added comment